ucb_prefect_tools/util.py

```diff
@@ -405,11 +401,9 @@ def _get_repo_info():
     return label, repo_short_name, branch_name
 
 
-def parse_docstring_fields(
-    function: callable, required_fields: list, optional_fields: list
-):
+def parse_docstring_fields(docstring: str):
```

Comment:
Rather than placing the `required_fields` and `optional_fields` within the function definition, should these be two parameters with specified defaults? Something like:

```suggestion
def parse_docstring_fields(
    docstring: str,
    required_fields: list[str] = ["schedule", "image_name", "main_params"],
    optional_fields: list[str] = ["tags", "source_systems", "sink_systems", "customer_contact"],
) -> dict[str, str]:
```